src/main/resources/templates/editmessage.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Edit a Gilf</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+</head>
+<body>
+<h1 th:text="#{updatemessage}"></h1>
+<form action = "#" th:action="@{/web/myprofile/editmessage(id=${messageId})}" th:object="${formData}" method="post">
+    <p><label for="title"  th:text="#{title} + ':'"/>
+        <input id="title" type="text" th:field="*{title}"/>
+    <p><label for="body" th:text="#{messagebody} + ':'"/>
+        <input id="body" type="text" th:field="*{messageBody}"/>
+    <p>
+        <label for="privacy" th:text="#{messageprivacy} + ':'"/>
+        <select id="privacy" th:field="*{privateMessage}">
+            <option th:value="true" th:text="#{private}"></option>
+            <option th:value="false" th:text="#{public}"></option>
+        </select>
+    <p th:if="${#fields.hasErrors('privateMessage')}"
+    th:text="${#strings.listJoin(#fields.errors('privateMessage'), ', ')}"></p>
```

Comment:
Same here for the reasons for not having the <p th:if="${#fields.hasErrors}" parts. I see you kept the hasErrors part for privatemessage, I'm guessing you forgot about removing it? Not a big deal, but it does not give any errormessage(not sure it can give a message either since it's a binary value).